### ISSUE TITLE
README.md 

### ISSUE THREAD

**Issue by ZZerog** [java-native/java-simple-serial-connector#20]
Hi,

will be great write better README.md.

For example: What is main goal of this fork?

**Comment by tresf** [comment]
Duplicate of #6.

The `README.md`  already offers more resources than the original `README.txt` from upstream.
- CI build status
- Download link
- Compilation instructions
- Code examples

We're open to more ideas.

> What is main goal of this fork?

It's in the description.

> Attempt to resurrect scream3r/java-simple-serial-connector

More specifically:
* Thread that started this branch: https://github.com/scream3r/java-simple-serial-connector/issues/132
* Fix Java 11 + Windows support, which is currently broken (fixed with our new build system)
* Eventually take over upstream maintenance of this project by obtaining special permission to transfer upstream from GitHub.

This information is not necessary for the `README.md` as once the transfer is complete, it will be self-explanatory.

We are in need of testers, especially for arm.  Work will be merged to `master` very soon, but we're making good progress on the `maven` and `maven-packaging` repos.
